hr_holidays_compute_days/models/hr_holidays.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# ©  2015 iDT LABS (http://www.@idtlabs.sl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from openerp.exceptions import ValidationError
+from dateutil.relativedelta import relativedelta
+import math
+
+
+class HrHolidays(models.Model):
+    _inherit = 'hr.holidays'
+
+    @api.model
+    def _check_date_helper(self, employee_id, date):
+        status_id = self.holiday_status_id.id or self.env.context.get(
+            'holiday_status_id',
+            False)
+        if employee_id and status_id:
+            employee = self.env['hr.employee'].browse(employee_id)
+            status = self.env['hr.holidays.status'].browse(status_id)
+            if (not employee.work_scheduled_on_day(
+                    fields.Date.from_string(date),
+                    public_holiday=status.exclude_public_holidays,
+                    schedule=status.exclude_rest_days)):
+                return False
+        return True
+
+    @api.multi
```

Comment:
@moylop260 The method is executed from onchange dates functions, and employee_id could be only in context, if i will putt depends on employee, the method will not be executed, that's why i choose to have employee ar as argument...